applications/crossbar/src/cb_context.erl

```diff
@@ -613,7 +613,13 @@ validate_request_data(Schema, Context, OnSuccess, OnFailure) ->
 -spec failed(context(), jesse_error:error_reasons()) -> context().
 -spec failed_error(jesse_error:error_reason(), context()) -> context().
 failed(Context, Errors) ->
-    lists:foldl(fun failed_error/2, set_resp_status(Context, 'error'), Errors).
+    lists:foldl(fun failed_error/2
+               ,set_resp_error_code(
+                  set_resp_status(Context, 'error')
+                  ,400
+                 )
```

Comment:
Do you mind using `setters/2`, before the fold? (I'm not a fan of function calls spanning 7 lines…)